velox/vector/DecodedVector.cpp

```diff
@@ -514,4 +516,35 @@ VectorPtr DecodedVector::wrap(
       rows.end(),
       std::move(data));
 }
+
+const uint64_t* DecodedVector::nulls() {
+  if (allNullsInitialized_) {
+    return allNulls_;
+  }
+
+  if (hasExtraNulls_) {
+    allNulls_ = nulls_;
+  } else if (!nulls_ || size_ == 0) {
+    allNulls_ = nullptr;
+  } else {
+    if (isIdentityMapping_) {
+      allNulls_ = nulls_;
+    } else if (isConstantMapping_) {
+      copiedNulls_.resize(0);
```

Comment:
why resize to 0 and then resize to nwords(size_) ? Why not do that in first place ?

Reply:
To make sure that all values are set to bits::kNull64.